addons/website/static/src/services/website_service.js

```diff
@@ -122,6 +122,7 @@ export const websiteService = {
                         // denominator of editable pages.
                         editable: !!document.getElementById('wrapwrap'),
                         viewXmlid: viewXmlid,
+                        direction: document.documentElement.querySelector('#wrapwrap.o_rtl') ? 'rtl' : 'ltr',
```

Comment:
I guess it's ok but is not there a better thing to do? relying on a class on #wrapwrap in the iframe seems fragile, can't the localization service be used somehow?

Reply:
Let's agree to keep this, if not easy to access inline services